README.md

```diff
@@ -10,6 +10,7 @@ This repository contains an implementation of the [AMWA Networked Media Open Spe
 - [AMWA IS-07 NMOS Event & Tally Specification](https://specs.amwa.tv/is-07/)
 - [AMWA IS-08 NMOS Audio Channel Mapping Specification](https://specs.amwa.tv/is-08/)
 - [AMWA IS-09 NMOS System Parameters Specification](https://specs.amwa.tv/is-09/) (originally defined in JT-NM TR-1001-1:2018 Annex A)
+- [AMWA IS-10 NMOS Authorization Specification](https://specs.amwa.tv/is-10/)
```

Comment:
SHould be not add AMWA BCP-003-02 to the list also?

Reply:
yeap